angular-client/src/utils/enumerations/identifier-data-type.ts

```diff
@@ -23,5 +28,13 @@ export enum IdentifierDataType {
   ACCELERATION = 'Acceleration',
   CHARGE_CURRENT_LIMIT = 'Pack-CCL',
   DISCHARGE_CURRENT_LIMIT = 'Pack-DCL',
-  XYZAccel = 'XYZAcceleration'
+  XYZAccel = 'XYZAcceleration',
+  STATUS = 'Status-Balancing',
+  BMS_MODE = 'Status-State',
+  VOLTS_HIGH = 'Cells-Volts_High_Value',
+  VOLTS_LOW = 'Cells-Volts_Low_Value',
+  CHARGING = 'Charging-Control',
```

Comment:
if this is a boolean, should be IS_CHARGING

Reply:
not sure. On call I need to better understand how data is passed in and how to read embedded yaml. But I think it should be.